src/content/docs/dropins/payment-services/containers/credit-card.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: CreditCard container
+description: Configure the CreditCard container to manage and display the available credit card fields during a checkout.
```

Comment:
> Configure the CreditCard container to manage and display the available [...] fields [...]

When reading this, I get the impression that the `CreditCard` container renders those fields that are "available". This seems confusing as there is no such thing as an "available field". Instead, it's all or nothing: either the credit card _form_ is available (with its three inputs: number+expdate+cvv), or it is not.

```suggestion
description: Use the CreditCard container to render a credit card form during checkout.
```